testworld/park.go

```diff
@@ -248,3 +253,21 @@ func (h *host) p2pURL() (string, error) {
 	}
 	return fmt.Sprintf("/ip4/127.0.0.1/tcp/%d/ipfs/%s", h.p2pPort, lastB58Key), nil
 }
+
+type hostTestSuite struct {
+	name   string
+	host   *host
+	id     identity.CentID
+	expect *httpexpect.Expect
+}
+
+func getHostTestSuite(t *testing.T, name string) hostTestSuite {
```

Comment:
I would like to keep these kind of util functions that deal with host information as part of `hostManager`